core/src/banking_stage.rs

```diff
@@ -1138,7 +1377,7 @@ impl BankingStage {
 
             tracer_packet_stats.report(1000);
 
-            let recv_timeout = if !buffered_packet_batches.is_empty() {
+            let recv_timeout = if !unprocessed_transaction_storage.is_empty() {
```

Comment:
Might be good to add a comment here that the gossip thread will almost always wait 100ms, even though it's never doing any processing, since the vote transaction storage is most likely not empty